p11-kit/Makefile.am

```
@@ -607,6 +612,10 @@ mock_twelve_la_SOURCES = p11-kit/mock-module-ep10.c
 mock_twelve_la_LDFLAGS = $(mock_one_la_LDFLAGS)
 mock_twelve_la_LIBADD = $(mock_one_la_LIBADD)
 
+mock_thirteen_la_SOURCES = p11-kit/mock-module-ep11.c
+mock_thirteen_la_LDFLAGS = $(mock_one_la_LDFLAGS) $(LIBTASN1_CFLAGS)
```

Don't add CFLAGS to LDFLAGS:
```suggestion
mock_thirteen_la_LDFLAGS = $(mock_one_la_LDFLAGS)
mock_thirteen_la_CFLAGS = $(LIBTASN1_CFLAGS)
```